\documentclass[titlepage,landscape]{seminar}
\usepackage{url}
\usepackage{graphicx}
\usepackage{hyperref}
\usepackage{epstopdf}
\usepackage{slides}

\newcommand{\frack}{\frac{1}{k}}
\newcommand{\quarter}{\frac{1}{4}}

\begin{document}

\myslide{
\begin{center}
\resizebox{\textwidth}{!}{\includegraphics{coalescent-figure.eps}}
\end{center}
}

\myslide{
\heading{Kingman's coalescent (two allele copies)}

\[
\mbox{P}(\hbox{two alleles ibd}) = \frac{1}{2N_e^{(f)}}
\]
\vfill
\[
\mbox{P}(\hbox{coalescent at time $t$} = \left(1 - \frac{1}{2N_e}\right)^{t-1}\frac{1}{2N_e}
\]
\vfill
\[
\hbox{Average time to coalescence of two alleles} = 2N_e
\]
}

\myslide{
\heading{Kingman's coalescent (multiple allele copies)}

\begin{eqnarray*}
m &=& \hbox{number of allele copies} \\
\hbox{number of allele copy pairs} &=& \frac{m(m-1)}{2} \\
\mbox{P}(\hbox{coalescent involving one pair of alleles}) &=& 
\left(\frac{m(m-1)}{2}\right)\left(\frac{1}{2N_e}\right) \\
\hbox{Average time to coalescent event} &=&
\left(\frac{2}{m(m-1)}\right)\left(2N_e\right) \\
&=& \frac{4N_e}{m(m-1)} 
\end{eqnarray*}
}

\myslide{
\heading{Kingman's coalescent (multiple allele copies)}

\begin{eqnarray*}
\hbox{Average time to coalescent event} &=& \frac{4N_e}{m(m-1)} 
\end{eqnarray*}
Now there are $m-1$ alleles
\begin{eqnarray*}
\hbox{Average time to next coalescent event} &=& \frac{4N_e}{(m-1)(m-2)} 
\end{eqnarray*}
\vfill
\begin{eqnarray*}
\bar t &=& \sum_{k=2}^m \frac{4N_e}{k(k-1)} \\
       && \mbox{TAMO} \\
       &=& 4N_e\left(1 - \frac{1}{m}\right) \\
       &\approx& 4N_e
\end{eqnarray*}
}

\myslide{
\heading{Mitochondrial Eve}

Data
\begin{itemize}

\item Mitochondrial DNA from 147 humans

\item Most recent common ancestor 200,000 years ago

\end{itemize}

Expectation
\begin{itemize}

\item All mitochondrial genomes share a single common ancestor $2N_e$
  generations ago.

\item Human generation time $\approx$ 20 years. 

\item 200,000 years $\approx$ 10,000 generations.

\item Coalescence of all mitochondria consistent with $N_e$ of 5000.

\end{itemize}
}

\myslide{
\heading{Coalescent in structured populations}

{\small
\begin{eqnarray*}
\bar t_0 &=& \mbox{average time to coalescence of allele copies from
             the same population} \\
\bar t_1 &=& \mbox{average time to coalescence of allele copies from
             different populations} \\
\bar t &=& \mbox{average time to coalescence of allele copies} \\
       &=& \frac{k(k-1)\bar t_1 + k\bar t_0}{k^2}
\end{eqnarray*}
}
\vfill
If mutation is rare,
\[
F_{st} = \frac{\bar t - \bar t_0}{\bar t} \quad .
\]
}

\myslide{
\heading{Divergence of DNA sequences}
{\small
\begin{eqnarray*}
q_{t+\Delta t} &=& (1 - \lambda\Delta t)^2q_t
                  + 2\left(1 - \lambda\Delta t\right)\left({1 \over
                           3}\lambda\Delta t\right)(1 - q_t)
                  + \hbox{o}(\Delta t^2) \\
               &=& (1 - 2\lambda\Delta t)q_t + \left({2 \over 3}\lambda
                                                    \Delta t\right)
                                              (1 - q_t) 
                  + \hbox{o}(\Delta t^2) \\
q_{t+\Delta t} - q_t &=& {2 \over 3}\lambda\Delta t - {8 \over
                           3}\lambda\Delta tq_t + \hbox{o}(\Delta t^2) \\
{q_{t+\Delta t} - q_t \over \Delta t} &=& {2 \over 3}\lambda - {8 \over
3}\lambda q_t + \hbox{o}(\Delta t) \\
\lim_{\Delta t\to 0}{q_{t+\Delta t} - q_t \over \Delta t}  = {dq_t \over dt} &=& {2 \over 3}\lambda - {8
\over 3}\lambda q_t \\
q_t &=& 1 - {3 \over 4}\left(1 - e^{-8\lambda t/3}\right) \\
\end{eqnarray*}
}
}

\myslide{
\heading{Rate variation among sites}
\begin{figure}
\begin{center}
\resizebox{\textheight}{!}{\includegraphics{gamma.eps}}
\end{center}
\caption{Examples of a gamma distribution.}\label{fig:asrv}
\end{figure}
}

\myslide{
\heading{Molecular clock}
\scriptsize
\begin{eqnarray*}
\mbox{\# of substitutions/generation} &=& (\mbox{\# of mutations/generation})\times(\mbox{probability
  of fixation}) \\
\lambda &=& \mu_0p_0 
\end{eqnarray*}
}

\myslide{
\heading{Molecular clock}
\scriptsize
\begin{eqnarray*}
\mbox{\# of substitutions/generation} &=& (\mbox{\# of mutations/generation})\times(\mbox{probability
  of fixation}) \\
\lambda &=& \mu_0p_0 \\
\mu_0 &=& 2N\mu 
\end{eqnarray*}
}

\myslide{
\heading{Molecular clock}
\scriptsize
\begin{eqnarray*}
\mbox{\# of substitutions/generation} &=& (\mbox{\# of mutations/generation})\times(\mbox{probability
  of fixation}) \\
\lambda &=& \mu_0p_0 \\
\mu_0 &=& 2N\mu \\
p_0 &=& \frac{1}{2N} 
\end{eqnarray*}
}

\myslide{
\heading{Molecular clock}
\scriptsize
\begin{eqnarray*}
\mbox{\# of substitutions/generation} &=& (\mbox{\# of mutations/generation})\times(\mbox{probability
  of fixation}) \\
\lambda &=& \mu_0p_0 \\
\mu_0 &=& 2N\mu \\
p_0 &=& \frac{1}{2N} \\
\lambda &=& \mu_0p_0 \\
        &=& (2N\mu)\left(\frac{1}{2N}\right) \\
        &=& \mu 
\end{eqnarray*}
}

\myslide{
\heading{Molecular clock and genetic diversity}
{\scriptsize
\begin{eqnarray*}
\mbox{\# of substitutions/generation} &=& (\mbox{\# of mutations/generation})\times(\mbox{probability
  of fixation}) \\
\lambda &=& \mu_0p_0 \\
\mu_0 &=& 2N\mu \\
p_0 &=& \frac{1}{2N} \\
\lambda &=& \mu_0p_0 \\
        &=& (2N\mu)\left(\frac{1}{2N}\right) \\
        &=& \mu 
\end{eqnarray*}
}\vfill
\[
f = \frac{1}{4N_e\mu + 1} \quad .
\]
}

\myslide{
\heading{Genetic code}
\tiny
\begin{center}
\begin{tabular}{llllllll}
\hline\hline
      & Amino &       & Amino &       & Amino &       & Amino \\
Codon & Acid  & Codon & Acid  & Codon & Acid  & Codon & Acid \\
\hline
UUU   & Phe   & UCU   & Ser   & UAU   & Tyr   & UGU   & Cys \\
UUC   & Phe   & UCC   & Ser   & UAC   & Tyr   & UGC   & Cys \\
UUA   & Leu   & UCA   & Ser   & UAA   & Stop  & UGA   & Stop \\
UUG   & Leu   & UCG   & Ser   & UAG   & Stop  & UGG   & Trp \\
      &       &       &       &       &       &       & \\
CUU   & Leu   & CCU   & Pro   & CAU   & His   & CGU   & Arg \\
CUC   & Leu   & CCC   & Pro   & CAC   & His   & CGC   & Arg \\
CUA   & Leu   & CCA   & Pro   & CAA   & Gln   & CGA   & Arg \\
CUG   & Leu   & CCG   & Pro   & CAG   & Gln   & CGG   & Arg \\
      &       &       &       &       &       &       & \\
AUU   & Ile   & ACU   & Thr   & AAU   & Asn   & AGU   & Ser \\
AUC   & Ile   & ACC   & Thr   & AAC   & Asn   & AGC   & Ser \\
AUA   & Ile   & ACA   & Thr   & AAA   & Lys   & AGA   & Arg \\
AUG   & Met   & ACG   & Thr   & AAG   & Lys   & AGG   & Arg \\
      &       &       &       &       &       &       & \\
GUU   & Val   & GCU   & Ala   & GAU   & Asp   & GGU   & Gly \\
GUC   & Val   & GCC   & Ala   & GAC   & Asp   & GGC   & Gly \\
GUA   & Val   & GCA   & Ala   & GAA   & Glu   & GGA   & Gly \\
GUG   & Val   & GCG   & Ala   & GAG   & Glu   & GGG   & Gly \\
\hline
\end{tabular}
\end{center}
}

\myslide{
\heading{Substitution rates}
\scriptsize
\begin{center}
\begin{tabular}{lcc}
\hline\hline
Locus     & Non-synonymous rate & Synonymous rate \\
\hline
Histone \\
\quad H4  & 0.00                & 3.94 \\
\quad H2  & 0.00                & 4.52 \\ 
Ribosomal proteins \\
\quad S17 & 0.06                & 2.69 \\
\quad S14 & 0.02                & 2.16 \\
Hemoglobins \& myoglobin \\
\quad $\alpha$-globin & 0.56    & 4.38 \\
\quad $\beta$-globin  & 0.78    & 2.58 \\
\quad Myoglobin       & 0.57    & 4.10 \\
Interferons \\
\quad $\gamma$  & 3.06          & 5.50 \\
\quad $\alpha$1 & 1.47          & 3.24 \\
\quad $\beta$1  & 2.38          & 5.33 \\
\hline
\end{tabular}
\end{center}
}

\end{document}

\end{document}

