\documentclass[titlepage,landscape]{seminar}
\usepackage{url}
\usepackage{graphicx}
\usepackage[pdftex]{color}
\usepackage{hyperref}
\usepackage{epstopdf}
\usepackage{slides}

\begin{document}

\myslide{
\begin{eqnarray*}
q_{t+\Delta t} &=& (1 - \lambda\Delta t)^2q_t
                  + 2\left(1 - \lambda\Delta t\right)\left({1 \over
                           3}\lambda\Delta t\right)(1 - q_t)
                  + \hbox{o}(\Delta t^2) \\
               &=& (1 - 2\lambda\Delta t)q_t + \left({2 \over 3}\lambda
                                                    \Delta t\right)
                                              (1 - q_t) 
                  + \hbox{o}(\Delta t^2)
\end{eqnarray*}
\begin{eqnarray*}
q_{t+\Delta t} - q_t &=& {2 \over 3}\lambda\Delta t - {8 \over
                           3}\lambda\Delta tq_t + \hbox{o}(\Delta t^2) \\
{q_{t+\Delta t} - q_t \over \Delta t} &=& {2 \over 3}\lambda - {8 \over
3}\lambda q_t + \hbox{o}(\Delta t) \\
\lim_{\Delta t\to 0}{q_{t+\Delta t} - q_t \over \Delta t}  = {dq_t \over dt} &=& {2 \over 3}\lambda - {8
\over 3}\lambda q_t \\
q_t &=& 1 - {3 \over 4}\left(1 - e^{-8\lambda t/3}\right) \\
\end{eqnarray*}
}

\myslide{
Number of substitutions: $d = 2\lambda t$
\vfil
\begin{eqnarray*}
q_t &=& 1 - {3 \over 4}\left(1 - e^{-8t/3}\right) \\
    &=& 1 - {3 \over 4}\left(1 - e^{-4d/3}\right) \\
d   &=& -{3 \over 4}\ln\left[1 - {4 \over 3}(1 - q_t)\right] 
\end{eqnarray*}
}

\myslide{
From Britten {\it PNAS\/} 99:13633-13635; 2002.

\begin{itemize}

\item 5 Chimpanzee genome sequences totaling 779,142bp

\item Aligned with ``the'' human genome

\item 1.41\% nucelotide sequence difference~(additional 3.91\%
  difference in idels)

\end{itemize}

\begin{eqnarray*}
d &=& -{3 \over 4}\ln\left(1 - {4 \over 3}(0.0141)\right) \\
  &=& -{3 \over 4}\ln(0.9812) \\
  &=& 0.0142
\end{eqnarray*}

For ``small'' amounts of sequence divergence $d \approx 1-q_t$.
}

\myslide{
From Liu et al. {\it Genome Research\/} 13:358-368; 2003.

\begin{itemize}

\item 623,139 aligned nucleotides between humans and lemur

\item 19.4\% nucleotide sequence difference

\end{itemize}

\begin{eqnarray*}
d &=& -{3 \over 4}\ln\left(1 - {4 \over 3}(0.194)\right) \\
  &=& -{3 \over 4}\ln(0.7413) \\
  &=& 0.22
\end{eqnarray*}

For ``large'' amounts of sequence divergence the number of
substitutions distinguishing two sequences is much larger than the
observed number of differences.
}

\myslide{
Suppose two sequences have been diverging for a very, very long time.

Then the probability that they match at any given nucleotide will be
about 1/4.

\begin{eqnarray*}
d &=& -{3 \over 4}\ln\left(1 - \left({4 \over 3}\right)\left(1- {1
      \over 4}\right)\right) \\
  &=& -{3 \over 4}\ln\left(1 - \left({4 \over 3}\right)\left({3 \over
        4}\right)\right) \\
  &=& -{3 \over 4}\ln(0) \\
  &=& \infty
\end{eqnarray*}
}

\myslide{
\begin{center}
\resizebox{\textwidth}{!}{\includegraphics{gamma.eps}}
\end{center}
}

\myslide{
\begin{eqnarray*}
\mbox{\# of substitutions/generation} &=& (\mbox{\# of
  mutations/generation}) \\
                                      && \times(\mbox{probability
  of fixation}) \\
\lambda &=& \mu_0p_0 
\end{eqnarray*}
\vfill
\[
\mu_0 = 2N\mu
\]
\vfil
\[
p_0 = \frac{1}{2N}
\]
\vfill
\[
\lambda = 2N\mu\left(\frac{1}{2N}\right) = \mu 
\]
}

\myslide{
\[
f = \frac{1}{4N_e\mu + 1}
\]
\vfill
\[
H = \frac{4N_e}{4N_e\mu + 1}
\]
\vfill
}

\myslide{\tiny
\begin{center}
\begin{tabular}{llllllll}
\hline\hline
      & Amino &       & Amino &       & Amino &       & Amino \\
Codon & Acid  & Codon & Acid  & Codon & Acid  & Codon & Acid \\
\hline
UUU   & Phe   & UCU   & Ser   & UAU   & Tyr   & UGU   & Cys \\
UUC   & Phe   & UCC   & Ser   & UAC   & Tyr   & UGC   & Cys \\
UUA   & Leu   & UCA   & Ser   & UAA   & Stop  & UGA   & Stop \\
UUG   & Leu   & UCG   & Ser   & UAG   & Stop  & UGG   & Trp \\
      &       &       &       &       &       &       & \\
CUU   & Leu   & CCU   & Pro   & CAU   & His   & CGU   & Arg \\
CUC   & Leu   & CCC   & Pro   & CAC   & His   & CGC   & Arg \\
CUA   & Leu   & CCA   & Pro   & CAA   & Gln   & CGA   & Arg \\
CUG   & Leu   & CCG   & Pro   & CAG   & Gln   & CGG   & Arg \\
      &       &       &       &       &       &       & \\
AUU   & Ile   & ACU   & Thr   & AAU   & Asn   & AGU   & Ser \\
AUC   & Ile   & ACC   & Thr   & AAC   & Asn   & AGC   & Ser \\
AUA   & Ile   & ACA   & Thr   & AAA   & Lys   & AGA   & Arg \\
AUG   & Met   & ACG   & Thr   & AAG   & Lys   & AGG   & Arg \\
      &       &       &       &       &       &       & \\
GUU   & Val   & GCU   & Ala   & GAU   & Asp   & GGU   & Gly \\
GUC   & Val   & GCC   & Ala   & GAC   & Asp   & GGC   & Gly \\
GUA   & Val   & GCA   & Ala   & GAA   & Glu   & GGA   & Gly \\
GUG   & Val   & GCG   & Ala   & GAG   & Glu   & GGG   & Gly \\
\hline
\end{tabular}
\end{center}
}

\myslide{
\begin{center}
\begin{tabular}{lll}
\hline\hline
      & Amino & \\
Codon & Acid  & Redundancy \\
\hline
CCU   & Pro   & 4-fold \\
CCC \\
CCA \\
CCG \\
\hline
AAU   & Asn   & 2-fold \\
AAC \\
AAA   & Lys   & 2-fold \\
AAG \\
\hline
\end{tabular}
\end{center}
}

\myslide{\scriptsize
\begin{center}
\begin{tabular}{lcc}
\hline\hline
Locus     & Non-synonymous rate & Synonymous rate \\
\hline
Histone \\
\quad H4  & 0.00                & 3.94 \\
\quad H2  & 0.00                & 4.52 \\ 
Ribosomal proteins \\
\quad S17 & 0.06                & 2.69 \\
\quad S14 & 0.02                & 2.16 \\
Hemoglobins \& myoglobin \\
\quad $\alpha$-globin & 0.56    & 4.38 \\
\quad $\beta$-globin  & 0.78    & 2.58 \\
\quad Myoglobin       & 0.57    & 4.10 \\
Interferons \\
\quad $\gamma$  & 3.06          & 5.50 \\
\quad $\alpha$1 & 1.47          & 3.24 \\
\quad $\beta$1  & 2.38          & 5.33 \\
\hline
\end{tabular}
\end{center}
}

\end{document}

