\documentclass[titlepage,landscape]{seminar}
\usepackage{url}
\usepackage{graphicx}
\usepackage{hyperref}
\usepackage{epstopdf}
\usepackage{slides}

\newcommand{\frack}{\frac{1}{k}}
\newcommand{\quarter}{\frac{1}{4}}

\begin{document}

\myslide{
  \heading{{\it Wyeomyia smithii\/} ({\it COI\/})}

  \begin{itemize}

  \item 1176 bp segment

  \item 20 populations of {\it W. smithii}

    \item One population each of {\it W. mitchellii\/} and {\it
        W. vanduzeei\/} (as outgroups)

  \end{itemize}
}

\myslide{
\heading{{\it Wyeomyia smithii\/} ({\it COI\/})}
\begin{center}
\includegraphics[height=0.9\textheight]{wyeomyia-COI.eps}
\end{center}
}

\myslide{
\heading{{\it Wyeomyia smithii\/} (RAD-seq)}
\begin{center}
\includegraphics[height=0.9\textheight]{RAD-tag.eps}
\end{center}
}

\myslide{
  \heading{{\it Wyeomyia smithii\/} (RAD-seq)}

  \begin{itemize}

   \item 21 populations, pooled sample of six individuals in each
     population, diegested with SbfI (CCTGCA$|$GG)

   \item 27.5M sequences, 14.9M passed quality control, each
     population rerpesented by $711,702 \pm 85,779$ sequences.

   \item $20,868 \pm 1681$ stacks and $13,627 \pm 1177$ loci per
     population

   \item 3741 SNPs fixed within at least two populations and
     variable among them

  \end{itemize}
  
}



\myslide{
\heading{{\it Wyeomyia smithii\/} (RAD-seq)}
\begin{center}
\includegraphics[height=0.9\textheight]{wyeomyia-RAD.eps}
\end{center}
}

\myslide{
  \heading{Nucleotide diversity: sample configuration at a homozygous site}

  \noindent Sample configuration
  \[
    (n_1, n_2, n_3, n_4) = \mbox{Number of }(A,C,G,T)\mbox{ in sample}
  \]
  \vfill
\[
P(n_1,n_2,n_3,n_4|\mbox{homozygous A},\epsilon)
=
{n \choose n_1}(1-\epsilon)^{n_1}\epsilon^{n-n_1}
\]
\vfill
\[
P(n_1,n_2,n_3,n_4|\mbox{homozygous},\epsilon)
=
\sum_{i=1}^4 \left(\frac{p_i^2}{\sum_{j=1}^4p_j^2}\right) 
{n \choose n_i}(1-\epsilon)^{n_i}\epsilon^{n-n_i}
\]
}

\myslide{
\heading{Nucleotide diversity: sample configuration at a heterozygous site}
\begin{eqnarray*}
  k_1 &=& \mbox{number of reads from chromosome 1} \\
  k_2 &=& \mbox{number of reads from chromosome 2} \\
P(k_1,k_2)
&=&
{n \choose k_1}\left(\frac{1}{2}\right)^{k_1}
               \left(\frac{1}{2}\right)^{k_2}
\\
&=&
{n \choose k_1}\left(\frac{1}{2}\right)^n
\end{eqnarray*}
}

\myslide{
\heading{Nucleotide diversity: sample configuration at a heterozygous site}
\vfill
{\footnotesize
  \begin{eqnarray*}
    x_i = \mbox{nucleotide on chromosome 1} \\
    x_j = \mbox{nucleotide on chromosome 2} \\
    \\
    \\
P(n_1,n_2,n_3,n_4|x_i,x_j,k_1,k_2) &=& \\
\sum_{l=1}^4\sum_{m=0}^{k_1}{k_1 \choose m}(1-\delta_{il})^m\delta_{il}^{k_1-m}
{k_2 \choose n_i-m}(1-\delta_{jl})^{n_1-m}\delta_{jl}^{k_2-(n_1-m)}
\end{eqnarray*}
}
where 
\[
\delta_{il} = \left\{\begin{array}{ll}
1-\epsilon & \mbox{if } i = l \\
\epsilon & \mbox{if } i \ne l \quad .
\end{array}
\right.
\]

}

\myslide{
\heading{Nucleotide diversity: sample configuration at a heterozygous site}
\[
P(n_1,n_2,n_3,n_4|x_i,x_j,\epsilon) =
P(n_1,n_2,n_3,n_4|x_i,x_j,k_1,k_2,\epsilon)P(k_1,k_2) \quad ,
\]
\vfill
{\footnotesize
\[
P(n_1,n_2,n_3,n_4|\mbox{heterozygous},\epsilon)
=
\sum_{i=1}^4\sum_{j\ne i}
\left(\frac{x_ix_j}{1-{\sum_{l=1}^4p_l^2}}\right) P(n_1,n_2,n_3,n_4|x_i,x_j,\epsilon) 
\]
}
}

\myslide{
\heading{Nucleotide diversity}
\[
\pi = \mbox{Probability that a site is heterozygous}
\]
\vfill
{\tiny
\[
P(n_1,n_2,n_3,n_4|\pi,\epsilon)
=
\pi P(n_1,n_2,n_3,n_4|\mbox{heterozygous},\epsilon)
+
(1-\pi)P(n_1,n_2,n_3,n_4|\mbox{homozygous},\epsilon) \quad .
\]
}
\vfill
\[
P(\mbox{data}|\pi,\epsilon) = \prod_{s=1}^S
P(n_1^{(s)},n_2^{(s)},n_3^{(s)},n_4^{(s)}|\pi,\epsilon)
\]
}

\myslide{
\heading{Nucleotide diversity}
\begin{center}
\begin{tabular}{lccc}
\hline\hline
Taxon & $4N_e\mu$ & $4N_e\mu$ (low coverage) & $\epsilon$ \\
\hline
{\it Cionia intestinalis} & 0.0111 & 0.012 & 0.00113 \\
{\it Daphnia pulex} & 0.0011 & 0.0012 & 0.00121 \\
\hline
\end{tabular}
\end{center}
}

\myslide{
\heading{PCA of population structure}
\begin{center}
\includegraphics[height=0.9\textheight]{fumagalli-PCA.eps}
\end{center}
}

\myslide{
\heading{Population genomics in Great Britain}

Data
\begin{itemize}

\item 2039 individuals with four grandparents born within 80km of one
  another, effectively studying alleles sampled from grandparents
  (ca. 1885). 

\item 6209 samples from 10 countries in continental Europe.

\item Autosomal SNPs genotyped in both samples~(ca. 500K). 

\end{itemize}
\vfill
Results
\begin{itemize}

\item Average pairwise $F_{ST}$: 0.0007

\item Maximum pairwise $F_{ST}$: 0.003

\end{itemize}
}

\myslide{
\heading{Population genomics in Great Britain}

\begin{center}
\includegraphics[height=0.9\textheight]{fine-structure-britain.eps}
\end{center}
}

\myslide{
\heading{Population genomics in Great Britain}

\begin{center}
\includegraphics[height=0.9\textheight]{UK-Europe.eps}
\end{center}
}

\end{document}
